deploy/olm-catalog/3scale-operator/3scale-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: 3scale-operator.v0.3.0
+  namespace: placeholder
+  annotations:
+    repository: https://github.com/3scale/3scale-operator
```

Comment:
Should this be a specific stable branch?

Reply:
`repository` should not reference specific stable branch

Reply:
oops that's true :+1: